code/modules/mob/living/basic/jungle/venus_human_trap.dm

```diff
@@ -161,76 +177,73 @@
 	lighting_cutoff_green = 35
 	lighting_cutoff_blue = 20
 	faction = list(FACTION_HOSTILE,FACTION_VINES,FACTION_PLANTS)
+	gold_core_spawnable = NO_SPAWN
 	initial_language_holder = /datum/language_holder/venus
 	unique_name = TRUE
-	/// A list of all the plant's vines
-	var/list/vines = list()
-	/// The maximum amount of vines a plant can have at one time
-	var/max_vines = 4
-	/// How far away a plant can attach a vine to something
-	var/vine_grab_distance = 5
+	speed = 1.2
+	melee_attack_cooldown = 1 SECONDS
+	ai_controller = /datum/ai_controller/basic_controller/human_trap
 	/// Whether or not this plant is ghost possessable
 	var/playable_plant = TRUE
 
-/mob/living/simple_animal/hostile/venus_human_trap/Life(seconds_per_tick = SSMOBS_DT, times_fired)
+/mob/living/basic/venus_human_trap/Initialize(mapload)
 	. = ..()
-	pull_vines()
+	AddElement(/datum/element/lifesteal, 5)
+	var/datum/action/cooldown/mob_cooldown/vine_tangle/tangle = new(src)
```

Comment:
If this was previously in `OpenFire` you should make sure that you trigger this ability in `RangedAttack` so that players can still use it by clicking far away without needing to toggle the ability.